README.md

```diff
@@ -30,6 +30,7 @@ local plugins = {
 	{ url = 'erf/vis-cursors', file = 'init' },
 	{ url = 'erf/vis-highlight', file = 'init', alias = 'hi' },
 	{ url = 'https://github.com/erf/vis-test.git', file = 'init', branch = 'other' },
+	{ url = 'https://github.com/samlwood/vis-gruvbox.git', file = 'init', theme = true },
```

Comment:
we dont need the `file = 'init'` here. you can also remove `https://github.com/` host, as it defaults to github

Reply:
> Should we add an option to `set theme` on startup?

Not really because vis already has a version to run a command at start with
```Lua
vis.events.subscribe(vis.events.INIT, function()
   vis:command('set theme mytheme')
end)

Reply:
I know, but i was thinking we could add a flag to the config, so we could do `set theme` for you, but let's not do it now